processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -16,7 +16,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.mapstruct.ap.internal.naming;
+package org.mapstruct.ap.spi;
```

Comment:
I think we didn't feel like exposing that default implementation to users as it will make it more difficult to change it in the future. But if you guys think it's ready for that, we can do it.